legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -65,7 +65,7 @@
     private static final Random RANDOM = new Random();
     private static final Encoder RANDOM_ID_ENCODER = Base64.getUrlEncoder().withoutPadding();
 
-    private static final Pattern PACKAGEABLE_ELEMENT_PATH_PATTERN = Pattern.compile("^\\w++(::\\w++)*+$");
+    private static final Pattern PACKAGEABLE_ELEMENT_PATH_PATTERN = Pattern.compile("^\\w++(::\\w++)*+(\\w*+\\$\\w++\\$\\w*+)*+$");
```

Comment:
Either we should tighten this up so that we're only allowing $ in the patterns we use for function names, or we should simplify it to allow $ to be used freely in the the name. The latter option would be something like this:
```suggestion
    private static final Pattern PACKAGEABLE_ELEMENT_PATH_PATTERN = Pattern.compile("^\\w++(::\\w++)*+[\\w$]*+$");
```

Reply:
Updated the code based on the latter option.